docs/cmdline-opts/fail.d

```diff
@@ -1,15 +1,15 @@
 Long: fail
 Short: f
 Protocols: HTTP
-Help: Fail silently (no output at all) on HTTP errors
+Help: Fail fast with no output on HTTP errors
 See-also: fail-with-body
 Category: important http
 Example: --fail $URL
 Added: 4.0
 ---
-Fail silently (no output at all) on server errors. This is mostly done to
-enable scripts etc to better deal with failed attempts. In normal cases
-when an HTTP server fails to deliver a document, it returns an HTML document
+Fail fast with no output at all on server errors. This is mostly done to
```

Comment:
How about s/done/useful/ here?

Reply:
Totally reasonable!

Reply:
I changed some more words too. Check the update!